test/resources/config/query.bal

```diff
@@ -30,4 +30,12 @@ function testFunction() {
             where memberAge > minMemberAge
             select memberAge).pop() >  minLeaderAge
         select leader;
+
+    var orders = [{orderCategory: "A", price: 123, id: 1}, {orderCategory: "B", price: 123, id: 2}];
+
+    var _  = from var {orderCategory, price} in orders 
+            group by orderCategory where orderCategory == "A" order by orderCategory limit 1 select sum(price);
```

Comment:
Shall we add this one also for tests?

`var _  = from var {orderCategory, price} in orders 
            group by var x = orderCategory where x == "A" order by x limit 1 select sum(price);`

`orderCategory` is referred by another variable here.

Reply:
Fixed.